### PR TITLE
Add bigquery create and copy table examples

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

**Comment by jerjou** [comment]
What's the difference between the `cloud-client` and `api` subdirs?

**Comment by jerjou** [comment]
Oh wait. Never mind - I see now.
We should probably add to cloud-client/README, or explain the distinction in the bigquery/README (not necessarily in this PR)

**Comment by jonparrott** [comment]
@jerjou I'm likely going to do a bulk re-name of all folders soon `api` will become `api-client` so it'll make a bit more sense.

File: bigquery/cloud-client/snippets_test.py [hunk 1]
```diff
@@ -62,7 +62,42 @@ def test_list_rows(capsys):
...
+@pytest.fixture
+def temporary_table():
+    """Fixture that returns a factory for tables that do not yet exist and
+    will be automatically deleted after the test."""
+    bigquery_client = bigquery.Client()
+    dataset = bigquery_client.dataset(DATASET_ID)
+    tables = []
+
+    def factory(table_name):
+        new_table = dataset.table('test_create_table')
```
Comment (jerjou) [inline bigquery/cloud-client/snippets_test.py]: I assume you mean `dataset.table(table_name)` here?

Comment (jonparrott) [inline bigquery/cloud-client/snippets_test.py]: lol yes. Good catch, forgot to change when I refactored. Fixed.

**Comment by jerjou** [comment]
LGTM

**Comment by jonparrott** [comment]
Thanks @jerjou!

**Pull request merged**